### PR TITLE
Various Iran 2015 updates: adds F-5 FC2024 flyable, consistent Iran liveries across planeset

### PR DESCRIPTION
**Pull request by ColonelAkirNakesh**
Changes F1EQ to EE to make it flyable.
Add F-5 FC2024
Removed aircraft that don't have Iran liveries as part of core game (Hind, Su-25T) Change IL-76  to C-130 (which has Iran livery in core game) Change Reaper to Wingloong
Enforce Iran liveries for modules with possibility for non-Iran liveries appearing -- F-4E, F-5, MiG-21 and Mirage F1

### CODE REVIEW

File: resources/factions/iran_2015.json [hunk 1]
```diff
@@ -7,22 +7,17 @@
...
-    "Mi-24V Hind-E",
-    "Mi-24P Hind-F",
-    "Mi-28N Havoc",
+    "F-5E-3_FC",
+    "Mi-8MTV2 Hip",
     "MiG-21bis Fishbed-N",
     "MiG-29A Fulcrum-A",
-    "Mirage-F1EQ",
-    "Su-22M4 Fitter-K",
+    "Mirage-F1EE",
```
Comment (Raffson) [inline resources/factions/iran_2015.json:18]: Don't swap this one out, rather add it next to the EQ 😉

File: resources/factions/iran_2015.json [hunk 2]
```diff
@@ -7,22 +7,17 @@
...
-    "IL-76MD",
-    "Mi-24V Hind-E",
-    "Mi-24P Hind-F",
-    "Mi-28N Havoc",
+    "F-5E-3_FC",
+    "Mi-8MTV2 Hip",
     "MiG-21bis Fishbed-N",
     "MiG-29A Fulcrum-A",
-    "Mirage-F1EQ",
-    "Su-22M4 Fitter-K",
```
Comment (Raffson) [inline resources/factions/iran_2015.json:22]: Why did the fitter get removed exactly? 😅

**Review by Raffson — CHANGES_REQUESTED** [review]
I see many aircraft got removed, but why? 😅

**Comment by ColonelAkirNakesh** [comment]
Ah I thought it would be good to clean out the generic airplane liveries
that don't have the Iran liveries, but if that was too aggressive is it
possible to just revert the changes, add the livery enforcement and FC3 F-5
and add the Mirage F1EE as you mentioned or should I start a new request?
Thanks and sorry for being a noob haha

**Comment by Raffson** [comment]
No worries, you can still revert the deleted aircraft in the same PR 😉

**Comment by ColonelAkirNakesh** [comment]
I think I'll kill this one off and try to make actual squadrons instead of
livery overrides for the Iranian modules that don't have em :)

**Pull request closed without merging**